sdk/cosmos/azure-cosmos/azure/cosmos/aio/container.py

```diff
@@ -198,16 +199,20 @@ async def create_item(
 
     @distributed_trace_async
     async def read_item(
-        self,
-        item,  # type: Union[str, Dict[str, Any]]
-        partition_key,  # type: Any
-        **kwargs  # type: Any
+            self,
+            item,  # type: Union[str, Dict[str, Any]]
+            partition_key,  # type: Any
+            max_integrated_cache_staleness_in_ms=None,  # type: Optional[int],
```

Comment:
@annatisch - is this python style of adding new parameters?
What if in the future, there are 10 more of these? Would we keep adding them here?
I am assuming, this is not a breaking change in python, and won't be in future if we keep doing this. Am I right?

Reply:
Yes and no.
We would keep adding keyword arguments. And would keep doing so in future.
However unlike here - they would need to be keyword only (otherwise it _can_ lead to future breaking changes). 
@simorenoh in this case it will either need to be rolled into kwargs (probably advisable given it's status), or use the `*` syntax to mark it as keyword-only.